packages/layout/src/node/createInstances.js

```diff
@@ -16,7 +21,7 @@ const isFragment = value =>
  * @param {Object} React element
  * @returns {Array} parsed react elements
  */
-const createInstances = element => {
+const createInstances = async element => {
```

Comment:
I don't think this is correct architectural-wise. We are mixing concepts here. `createInstances` should not fetch images, that's done separately on another layer. Can we preserve that?

Reply:
I'm not very familiar with this project and was just looking for a quick solution. Perhaps this issue would be better addressed by someone more acquainted with the codebase.

Reply:
I suggested 2 possible approaches here https://github.com/diegomura/react-pdf/issues/1369#issuecomment-1087910517
Do they make sense @diegomura?